resources/tickets.py

```diff
@@ -87,7 +87,11 @@ class Tickets(Resource):
 
     @jwt_required
     def get(self):
-        return {'tickets': [ticket.json() for ticket in TicketModel.query.all()]}
+        data = Tickets.parser.parse_args()
+        if data["tenant"]:
+            return {'Tickets': [ticket.json() for ticket in TicketModel.find_by_tenantID(data["tenant"])]}
+        else:
+            return {'Tickets': [ticket.json() for ticket in TicketModel.query.all()]}
```

Comment:
`{'Tickets':` <- These capital `T`'s are causing a failure. I'm not opposed to using a capitalized version, however I think we have a convention of using lowercase everywhere.

Reply:
Thanks for catching this, it was a typo, sorry I missed the failing test.